web/src/ts/store/auth/reducer.ts

```diff
@@ -0,0 +1,41 @@
+import { UserAction, AuthState, UserActionTypes } from '.'
+
+const initialState: AuthState = {
+  user: {
+    createdAt: null,
+    email: null,
+    id: null,
+    username: null,
+  },
```

Comment:
By default it should be
```suggestion
  user: {},
```